templates/settings.py.erb

```diff
@@ -63,6 +63,9 @@ DEFAULT_ENVIRONMENT = '<%= @default_environment %>'
 <% if @reports_count -%>
 REPORTS_COUNT = <%= @reports_count %>
 <% end -%>
+<% if @graph_facts -%>
+GRAPH_FACTS = '<%= @graph_facts.sort.uniq.join(',') %>'
```

Comment:
please add a unit test for the change with a string as input and one with an array. We need to ensure the sort.uniq.join construct works with strings as well.